recipe.json

```diff
@@ -0,0 +1,27 @@
+{
+    "RecipeFormatVersion": "2020-01-25",
+    "ComponentName": "COMPONENT_NAME",
+    "ComponentVersion": "COMPONENT_VERSION",
+    "ComponentType": "aws.greengrass.plugin",
+    "ComponentDescription": "test for log manager",
+    "ComponentPublisher": "COMPONENT_PUBLISHER",
+    "ComponentDependencies": {
```

Comment:
Don't commit this file here - good for testing but we don't need to add this as part of these changes